cypress-tests/cypress/constants/texts/textInput.js

```diff
@@ -0,0 +1,12 @@
+export const textInputText = {
+  customValidation: (name, message,) => {
+    return ["{{",`components.${name}.value ? true : '${message}'}}`,];
+  },
+
+  defaultWidgetName: "textinput1",
+  textInputRegex: "^[A-Z]*$",
+  textMinimumLength: "4",
```

Comment:
Can we add some limited random value for max and min length?
eg: [1-10]

Reply:
Yeah, sure we can add random values for min and max length. Updated on the latest commit